ext/storage_sources/gcp_store/README.md

```diff
@@ -16,18 +16,21 @@ This section describes how to build WiredTiger with the GCP extension enabled.
 * clang 6.0.0 or higher
 
 ### Building
-There is currently only 1 way to build WiredTiger with the GCP extension:
-This way manages the GCP SDK dependency as an external project. This method will download the
-GCP SDK, link to WiredTiger's build system, and build the extension.
+There are two ways to build WiredTiger with the GCP extension:
+
+1. Using a system installation of the GCP SDK. Information about how to perform a system install 
+from the SDK can be found [here](https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md.).
```

Comment:
Grammar: "from the SDK" -> "to the SDK"

Reply:
[3433d6a](https://github.com/wiredtiger/wiredtiger/pull/9000/commits/3433d6a7002b208627e313092587c0b6cf019f25)